seed/serialize.py

```diff
@@ -212,11 +204,12 @@ def make_field_trial_testing_config(seed, version_string, channel_string):
 
 
 if __name__ == "__main__":
-    print("Load seed.json")
     parser = argparse.ArgumentParser()
-    parser.add_argument('seed_path', help='json seed file to process')
     parser.add_argument(
-      '--fieldtrial-testing-config-path', type=str,
+      'seed_path', type=argparse.FileType('r'), nargs='?', default=sys.stdin,
```

Comment:
can you add `--seed-path` for consistency with other args and keep `seed_path` as a deprecated option?

Reply:
```suggestion
      'seed-path', type=argparse.FileType('r'), nargs='?', default=sys.stdin,
```

Reply:
Sadly for positional arguments the first param is the `dest`. If we use `seed-path` we have to use `vars(args)['seed-path'` : https://stackoverflow.com/questions/12834785/having-options-in-argparse-with-a-dash

Also the one parameter can't be used as positional & non-positional.
So IMHO the current code is acceptable. Also this can't be moved to separate .py file to make things clear.